src/components/FilterBarInline/FilterBarInline.mdx

```diff
@@ -0,0 +1,349 @@
+
+**Note:** Future DS updates may provide a standalone component to implement a
+UI container similar to what is used in these example.
+
+### Row Layout
+
+<Canvas of={FilterBarInlineStories.UIContainers} />
+
+<Source
+  code={`
+/* LIGHT MODE ONLY */
+// Add container for row layout
+<Box bg="ui.bg.default" p="inset.wide">
+  <FilterBarInline
+    layout="row"
+    renderChildren={() => {
+      // Filter components...
+    }}
+    // ...
+  />
+</Box>
+`}
+  language="jsx"
+/>
+
+<Source
+  code={`
+/* DARK MODE SUPPORT */
+// Import useColorModeValue hook
+import { useColorModeValue } from "@nypl/design-system-react-components";
+// ...
+// Use hook to set values for light and dark mode
+const containerBgColor = useColorModeValue(
+  "ui.bg.default",
+  "dark.ui.bg.default"
+);
+// ...
+// Add container for row layout
+<Box bg={containerBgColor} p="inset.wide">
+  <FilterBarInline
+    layout="row"
+    renderChildren={() => {
+      // Filter components...
+    }}
+    // ...
+  />
+</Box>
+; `}
+  language="jsx"
+/>
+
+### Column Layout
+
+<Canvas of={FilterBarInlineStories.ColumnLayout} />
+
+<Source
+  code={`
+/* LIGHT MODE ONLY */
+// Add container for column layout
+<Box bg="ui.bg.default" p="inset.default">
+  <FilterBarInline
+    layout="column"
+    renderChildren={() => {
+      // Filter components...
+    }}
+    // ...
+  />
+</Box>
+`}
+  language="jsx"
+/>
+
+<Source
+  code={`
+/* DARK MODE SUPPORT */
+// ...
+// Import useColorModeValue hook
+import { useColorModeValue } from "@nypl/design-system-react-components";
+// ...
+// Use hook to set values for light and dark mode
+const containerBgColor = useColorModeValue(
+  "ui.bg.default",
+  "dark.ui.bg.default"
+);
+// ...
+// Add container for column layout
+<Box bg={containerBgColor} p="inset.default">
+  <FilterBarInline
+    layout="column"
+    renderChildren={() => {
+      // Filter components...
+    }}
+    // ...
+  />
+</Box>;
+`}
+  language="jsx"
+/>
+
+## Passing Children
+
+<Canvas of={FilterBarInlineStories.MultipleChildren} />
+
+In order to control the styling and props passed to children form elements,
+the `FilterBarInline` component accepts `CheckboxGroup`, `MultiSelectGroup`,
+`RadioGroup`, `Checkbox`, `DatePicker`, `MultiSelect`, `Radio`, `Select`,
+`Slider`, `TextInput`, and `Toggle` children through its `renderChildren` render
+prop function.
+
+This callback prop function returns an object argument with three properties:
+`isBlockElement`, `layout`, and `width`.
+
+- The `isBlockElement` argument value should be passed to the `isBlockElement`
+  prop of `MultiSelect` children.
+- The `layout` argument value should be passed to the `layout` prop of
+  `CheckboxGroup`, `MultiSelectGroup`, and `RadioGroup` children.
+- The `width` argument value should be passed to the `multiSelectWidth` prop of
+  `MultiSelectGroup` children and the `width` prop of `MultiSelect` children.
+
+## Controlling State Using selectedItems, onClear, and onSubmit Props
```

Comment:
I really like the flow of this section, showing code and explaining it as it gets to the final component render.